src/opnsense/mvc/app/controllers/OPNsense/Firewall/Api/FilterController.php

```diff
@@ -88,11 +88,16 @@ public function searchRuleAction()
         /* filter logic for mvc rules */
         $filter_funct_mvc = function ($record) use ($categories, $interfaces, $show_all) {
             $is_cat = empty($categories) || array_intersect(explode(',', $record->categories), $categories);
+            $rule_interfaces = explode(',', (string)$record->interface);
```

Comment:
does explode() produce a valid entry with an empty interface "" here? If so why not use array_filter and avoid calling isEmpty later...